supplementary_style_guide/style_guidelines/grammar.adoc

```diff
@@ -83,5 +83,20 @@ As needed, adjust the conversational to _fairly conversational_ for an audience
 
 Red Hat Enterprise Linux 8 delivers a stable, secure, and consistent foundation across hybrid cloud deployments with the tools needed to deliver workloads faster with less effort.
 
+== Homographs
+A homograph is a word that is spelled the same as another word but has a different meaning, 
+often when the word is used as a different part of speech.
+Using homographs close together in a sentence or paragraph might confuse readers. 
+Therefore, when possible, avoid using homographs close to one another. 
+
+Homographs that might commonly appear in technical documentation include the following:
+
+* attribute
```

Comment:
How about adding the following >> **object** (objects are used a lot in OpenShift docs e.g _A custom resource is an object that extends the Kubernetes API or allows you to introduce your own API into a project or a cluster_)  .  **File** ( We use YAML files but file has two diff meanings >> Every file on the same disk must have a different name OR
We began to file out into the car park.)  Manifest (e.g. https://docs.openshift.com/container-platform/4.11/operators/operator_sdk/osdk-pkgman-to-bundle.html#osdk-migrating-pkgman_osdk-pkgman-to-bundle)

Reply:
I agree with adding **object** and will add it. But I would argue that sentences with **file** (verb) and **manifest** (verb) are unlikely to appear in technical docs. What do others think?

Reply:
crash?

Reply:
Does the verb/adjective second fit in this list?

Reply:
Does the verb/adjective second fit in this list?

Reply:
> Does the verb/adjective second fit in this list?

It's a homograph for sure, but imo both uses are not common in tech docs. I'm not sure how often we find "second" used a verb in technical writing. I'm trying to keep the list limited to homographs in which both uses of the word are somewhat common in technical docs.  Can you cite some uses where you've seen "second" used as a verb in tech docs? It's common in meeting minutes, for sure, ("I second that motion," e.g.) but outside of that, I don't really know how it'd be used in a technical context.

Reply:
This isn't going to be a comprehensive list, so I don't think we need to look for more examples to add. Just listing a few examples should be sufficient. I was almost thinking that the number of examples we have is pushing it (~5 would be good IMO). So I'd limit them to just a handful that we think is most common/likely.

Reply:
I agree @bergerhoffer. I've reduced the list to 6 of what I think might be the most common instances.